omod/src/main/webapp/fragments/patientsearch/patientSearchWidget.gsp

```diff
@@ -91,3 +91,10 @@
 </form>
 
 <div id="patient-search-results"></div>
+
+<% if (showRegisterPatientLink == "true") { %>
+<div>
+${ ui.message("coreapps.findPatient.registerPatient.label") }&nbsp;&nbsp;<a id="patient-search-register-patient" class="button" href="${ui.pageLink("registrationapp", "registerPatient", ["appId":"referenceapplication.registrationapp.registerPatient"])}">${ui.message("registrationapp.registration.label")}</a>
```

Comment:
I was referring to this discussion: https://talk.openmrs.org/t/how-to-make-the-reference-application-more-configurable/7632/29

It's not okay for the default behavior given by the coreapps module to refer to the referenceapplication module (and technically speaking, you can use the coreapps module without the registrationapp module, so even that reference is problematic).

Thinking about this a bit more, here's what I would do. Instead of controlling this by a global property, instead it should be controlled by an app configuration property called "showRegistrationLink" whose value is an application link.

Then, we'd do the same thing that's currently done with Register Patient, e.g. coreapps defines a basic version of the app, but the referenceapplication disables this and provides its own version that has the link to registration turned on.

Does that make sense?